README.md

```diff
@@ -162,9 +181,86 @@ specifies channel 0 which is connected to the thermometer.
     iex> volts = counts / 1023 * 3.3
     1.461290322580645
 
+Since many people coming to `elixir_ale` may be new to reading data sheets 
+and working with bitstrings, the following is a more detailed explanation.
```

Comment:
I almost thought that you were going to say "For more information on bitstrings, run `h <<>>` in IEx." or something similar. That might still be helpful somewhere else.